docker-compose.yml

```diff
@@ -110,3 +110,33 @@ services:
     environment:
       DD_REQUEST_DUMPER_FILE: dump.json
     command: sh -c "curl --output index.php https://raw.githubusercontent.com/DataDog/dd-trace-php/master/tests/request_replayer/index.php -q && php -S 0.0.0.0:80 index.php"
+
+  agent:
+    image: datadog/agent:latest
+    volumes:
+    - /var/run/docker.sock:/var/run/docker.sock:ro
+    - /proc/:/host/proc/:ro
+    - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
+    environment:
+    - DD_API_KEY=${DATADOG_API_KEY}
```

Comment:
Was this intentional to duplicate the agent container `ddagent_integration` here and pull the API key from the env vars instead of using the value `invalid_key_but_its_ok`? If so, cool, but wanted to double check. :)

Reply:
This can be used together with `apache_56_server` container to send real traces to our backend. I can drop this no problem (or move to a different file). But it may be useful to have some real server sending traces to our backend.

Reply:
Perfect! Just wanted to double check. :)